01-mathematical operations/operations.test.js

```diff
@@ -0,0 +1,199 @@
+const { add, subtract, divide , multiply } = require("./operations");
+
+describe("add function", () => {
+
+  test('Checks if the add function has a definition' , () => { 
+    expect(add).toBeDefined()
+  })
+
+  test("Positive Value ", () => {
+    expect(add(5, 3)).toBe(8);
+  });
+
+  test("Negative and Positive Value ", () => {
+    expect(add(-5, 3)).toBe(-2);
+  });
+  test("Positive and Negative Value ", () => {
+    expect(add(5, -3)).toBe(2);
+  });
```

Comment:
bele yox, yanawma funksionalin iwlediyini testiq etmekdi 
birde eyni tipli test yazma esas odu 13 ve ya 16 ci setirdeki testin ikisinden birini yazmisan

should sum correctly negative and positive numbers

Reply:
aha yaxşı  bu tip yazilişları silip push edirem indi